### DISCUSSION TITLE
Can we support a pure-CPU version installation?

### DISCUSSION THREAD

**Discussion by kaixuanliu** [alibaba/graphlearn-for-pytorch#25] [General]
Hi, as I do not have an environment with Nvidia GPU and CUDA, hence I cannot build GLTorch from source, even if I set `WITH_CUDA` option to 'OFF' in CMakeList.txt file. So could you help support installation  without cuda dependency? Thanks.

**Comment by yao-matrix** [comment]
@LiSu

**Comment by kaixuanliu** [comment]
Sorry, I figured out how to build CPU-only GLTorch just now. Maybe we can just adjust the README part. Close this discussion now.
